src/include/index/flat_pq_index.h

```diff
@@ -78,12 +78,12 @@ class flat_pq_index {
  public:
   // metadata
   uint64_t dimensions_{0};
-  size_t num_subspaces_{0};
-  size_t sub_dimensions_{0};
-  size_t bits_per_subspace_{8};
-  size_t num_clusters_{256};
-  float tol_ = 0.001;
-  size_t max_iter_ = 16;
+  uint64_t num_subspaces_{0};
```

Comment:
Should `num_subspaces_, sub_dimensions_` be `uint32_t`?

Reply:
Thanks, done.